gui/src/main/java/org/verapdf/cli/CliConstants.java

```diff
@@ -43,5 +43,45 @@ private CliConstants() {
 			MESS_PROC_STDIN_4
 	});
 
+	/**
+	 * All valid exit codes from veraPDF CLI
+	 * 
+	 * @author  <a href="mailto:carl@openpreservation.org">Carl Wilson</a>
+	 *          <a href="https://github.com/carlwilson">carlwilson AT github</a>
+	 *
+	 * @version 0.1
+	 */
+	public enum ExitCodes {
+		/** All files parsed and valid */
+		VALID(0, "All files validated."),
+		/** All files parsed, some invalid */
+		INVALID(1, "Invalid PDF/A file(s) found."),
+		/** Bad command line parameters */
+		BAD_PARAMS(2, "Invalid command line parameters."),
+		/** Out of Memory */
+		OOM(3, "Out of Java heap space (memory)."),
+		/** No files from passed list or directory */
+		NO_FILES(4, "No files to process."),
+		/** Java I/O Exception during processing */ 
+		IO_EXCEP(6, "I/O Exception while processing."),
```

Comment:
I would suggest to use IO_EXCEPTION enum name here (full name + similar to VERAPDF_EXCEPTION and JAXB_EXCEPTION)

Reply:
agreed and fixed